Example/Sources/Settings+UserDefaults.swift

```diff
@@ -31,12 +31,12 @@ extension UserDefaults {
     // MARK: - Mock Messages
     
     func setMockMessages(count: Int) {
-        set(count, forKey: "mockMessages")
+        set(count, forKey: messagesKey)
```

Comment:
```suggestion
        set(count, forKey: UserDefaults.messagesKey)
```

should fix the failing build